src/__tests__/to-be-disabled.tsx

```diff
@@ -79,6 +91,20 @@ describe('.toBeEnabled', () => {
       expect(() => expect(queryByTestId(name)).not.toBeEnabled()).toThrow();
     });
   });
+
+  test('handle editable prop for TextInput when true', () => {
```

Comment:
```suggestion
  test('handle editable prop for TextInput', () => {
```
you can remove the `when true` since you handle `undefined`, `true` and `false` in this test